data/changelogs/page-shield.yaml

```
@@ -0,0 +1,21 @@
+---
+link: /page-shield/changelog/
+productName: Page Shield
+productArea: Application security
+productAreaLink: /fundamentals/reference/changelog/security/
+entries:
+
+- publish_date: '2024-04-26'
+  title: Suggestions for the default directive
+  description: |-
+    When creating a Policy in the dashboard, default directive aggregates suggestions of monitored scripts and connections data, enabling defining default directive easier.
```

```suggestion
    When creating a policy in the dashboard, default directive aggregates suggestions of monitored scripts and connections data, enabling defining default directive easier.
```
I haven't been capitalizing "policy".